app/controllers/index.js

```diff
@@ -24,11 +27,18 @@ export const index = async (ctx) => {
         floor
         city
         headshot
+        team
       }
     }`)
   )
   state.set('teams', teams)
   state.set('cities', cities)
   state.set('members', members)
+  state.set('allMembers', members)
   ctx.render({ body: Index })
 }
+
+export const filterMembers = async (attrs) => {
+  state.set('curFilter', values(attrs)[0])
+  state.set('members', filter(state.get('allMembers'), attrs))
```

Comment:
This is the controller function that gets called when you click a filter. It updates the `members` property in our state tree which represents the list of visible members, as well as a `curFilter <String>` property that represents the value of the currently selected filter (used to highlight in purple below).